service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -89,6 +89,10 @@ protected boolean doRegister() {
 
     private void checkSchemaIdSet() {
         Microservice existMicroservice = srClient.getMicroservice(microservice.getServiceId());
+        if (existMicroservice == null) {
```

Comment:
only happened when the microservice been deleted after "srClient.getMicroserviceId".
if this happened, it's wrong to just log and return, because we need to reregister the microservice or just refused boot.